src/clj/vanilla/db/core.clj

```diff
@@ -150,23 +150,113 @@
 
   (get-services vanilla-db)
 
+  (delete-all-services! vanilla-db)
 
+  (delete-service! vanilla-db {:id "1000"})
+  (delete-service! vanilla-db {:id "4000"})
+  (delete-service! vanilla-db {:id "7000"})
 
 
 
 
 
-  (delete-all-services! vanilla-db)
+  (drop-services-table vanilla-db)
 
-  (delete-service! vanilla-db {:id "1000"})
-  (delete-service! vanilla-db {:id "4000"})
-  (delete-service! vanilla-db {:id "7000"})
+  ())
 
 
+; ***Layout table rich comment***
+(comment
 
+  (hugsql/def-db-fns "sql/queries.sql")
+  (hugsql/def-sqlvec-fns "sql/queries.sql")
 
+  (create-layout-table-sqlvec vanilla-db)
 
-  (drop-services-table vanilla-db)
+  (create-layout-table vanilla-db)
```

Comment:
i will be important to do ALL the database bootstrapping calls form ALL the recent PRs to be sure the database is built correctly rot support ALL features.

Lets make a big (do ...) the has all the creates and inserts for everything (services, layout, etc) to make it easier to find all the correct code, write the next update, and call all the functions that setup all the bootstrap data.

Reply:
Yes agreed, can we add a 'do' to either that defrecord or setup-database functions there already?  Or write a new one?  I suppose this kinda ties into that "initialize" namespace we were thinking about early in this task.  I committed one change to vanilla_db with an empty layout table so for dev it works, but need a long term solution

Reply:
I'm thinking we write a function to do everything (throw away the old tables, create new, fill everything that needs filling) and then NOT CALL IT form within the app. Just leave it so we can hit it from the repl.

Reply:
Done